src/components/book.js

```diff
@@ -0,0 +1,32 @@
+import PropTypes from 'prop-types';
+
+const Book = (props) => {
+  const { title, author } = props;
```

Comment:
:orange_circle: [OPTIONAL]
The Book component should receive its props directly as function arguments, like this:
```
const Book = ({ title, author }) => {
  // Your component code here
}
```